services/cd-service/pkg/repository/repository.go

```diff
@@ -252,6 +253,13 @@ func (r *repository) ApplyTransformers(ctx context.Context, transformers ...Tran
 		Email: r.config.CommitterEmail,
 		When:  time.Now(),
 	}
+	// Add Author to Commit MSG
+	actionAuthor := auth.Extract(ctx)
+	if actionAuthor != nil {
+		authMsg := fmt.Sprintf("Authored by:\nEmail: %s\nUsername: %s\n", actionAuthor.Email, actionAuthor.Username)
```

Comment:
Please set the actual git author in CreateCommitFromIds ( it's the 2nd parameter ) because that data is actually transfered to the frontend.

Reply:
OK, that is much better. thanks!